homeassistant/components/device_tracker/strings.json

```
@@ -15,6 +15,29 @@
       "state": {
         "home": "[%key:common::state::home%]",
         "not_home": "[%key:common::state::not_home%]"
+      },
+      "state_attributes": {
+        "battery": {
+          "name": "Battery"
+        },
+        "gps_accuracy": {
+          "name": "GPS Accuracy"
```

Actually spotted this one just now... oops :)

```suggestion
          "name": "GPS accuracy"
```